generators/server/templates/src/main/java/package/config/_ExcludedFromComponentScan.java

```diff
@@ -0,0 +1,9 @@
+package <%=packageName%>.config;
+
+/**
+ * a simple annotation, to be used for easy exclusion from component scan
+ *
+ * @author David Steiman
+ */
```

Comment:
I'm not a big fan of this annotation: it should be a standard Spring Boot annotation (need to check if this exists), or we can just exclude classes as usual. And it's only used once here.

It could be a good idea (if it doesn't already exists in Spring Boot), but then we would need to use it elsewhere in our code (we have other exclusions), and that should be another PR as it's a different issue.